pkg/console/install_panels.go

```diff
@@ -714,7 +754,19 @@ func addAskCreatePanel(c *Console) error {
 			if err != nil {
 				return err
 			}
+
+			if alreadyInstalled {
+				// need to wipe the Mode value before we set panel
+				// needed to ensure that value lookup fails as install
+				// is not available in the option in this case
+				c.config.Install.Mode = ""
+			}
+
 			c.config.Install.Mode = selected
+			if selected == config.ModeInstall {
+				installModeOnly = true
```

Comment:
Does installModeOnly need to reset to false if selected != config.ModeInstall ?

The user may select the install mode, return by ESC, and then select create mode

Reply:
makes sense.. let me make the change.